app/util/searchUtils.js

```diff
@@ -311,11 +330,55 @@ export function getGeocodingResult(
     opts = { ...opts, sources };
   }
 
-  return getJson(config.URL.PELIAS, opts).then(response =>
-    mapPeliasModality(response.features, config),
+  if (!config.URL.GEOCODING_BASE_URL.includes('digitransit')) {
```

Comment:
This seems a bit fragile? At least the check should be lifted to its own function since it's used a few times